special.c

```diff
@@ -79,12 +79,12 @@ static request_t *zero_read_op(object_t *o, request_t *r)
 
 static request_t *random_read_op(object_t *o, request_t *r)
 {
-	int randbuff[16];
-	size_t i, limit, chunk, len = r->msg.o.size;
+	size_t len = r->msg.o.size;
 
 	while (len > 0) {
-		chunk = (len > sizeof(randbuff)) ? sizeof(randbuff) : len;
-		limit = (chunk + sizeof(*randbuff) - 1) / sizeof(*randbuff);
+		int randbuff[16];
+		size_t chunk = (len > sizeof(randbuff)) ? sizeof(randbuff) : len;
+		size_t limit = (chunk + sizeof(*randbuff) - 1) / sizeof(*randbuff);
```

Comment:
Declarations no longer need to be at the top only in this repo or in all repos?

Reply:
Yup, please take a look at the updated coding guidelines

Reply:
so happy

Reply:
But excluding kernel, mind you